NEWS.md

```diff
@@ -0,0 +1,53 @@
+# Changes by Version
+
+## [Unreleased]
+
+### Added
+
+* Add `tmpdir` parameter to specify a custom temp directory (#230)
```

Comment:
If we were to use GitHub's releases feature instead of add a file to the repo ourselves, it automatically links issues there.